bin/cloudgov/delete

```diff
@@ -1,7 +1,23 @@
-#!/bin/sh
-#
-# This script will attempt to clean up the drupal install in cloud.gov
-#
+#!/bin/bash
+
+usage="
+$0: Attempts to clean up the drupal install in cloud.gov. (Probably currently broken)
```

Comment:
The phrase "Probably currently broken" along with the fact that this script's name is "delete" is a little worrying. How about adding couple lines after the usage to point that out and exit, e.g., 

echo "This script hasn't been reviewed in a long time and is probably broken. Please examine it and edit before running it." 
exit 1